src/test/java/seedu/address/logic/parser/AddressBookParserTest.java

```diff
@@ -45,6 +46,18 @@ public class AddressBookParserTest {
 
     private final AddressBookParser parser = new AddressBookParser();
 
+    @Test
+    public void parseCommand_exceedCharLimit_throwsParseException() {
```

Comment:
I like how you tested the implemented code by using a character length of 10,000.